libraries/chain/scorum_evaluator.cpp

```diff
@@ -97,24 +70,26 @@ struct strcmp_equal
 
 void witness_update_evaluator::do_apply(const witness_update_operation& o)
 {
-    _db.get_account(o.owner); // verify owner exists
+    dbs_account& accountService = _db.obtain_service<dbs_account>();
```

Comment:
I would prefer to use same variable naming conversion around the project. That means that variables should be in lower case and splitted with _.